content/en/blog/2023/kubecon-eu.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Join us for OpenTelemetry Talks and Activities at Kubecon EU 2023
+linkTitle: Kubecon EU '23
+date: 2023-03-30
+spelling:
+  cSpell:ignore Severin Neumann Pavol Loffay Kowall Aiven Vider Xiaochun Alolita
+  Sharma Dyla Benedikt Bongartz Oliveira Pathak
+author: '[Severin Neumann](https://github.com/svrnm)'
+---
+
+The OpenTelemetry project maintainers, and members of the governance committee
+and technical committee are excited to be at [Kubecon EU][] in Amsterdam from
+April 18 - 21, 2023!
+
+You can meet us at the [project booth][], listen to our talks or learn with us
+during workshops. We are looking forward to hear from you, how you are using
+OpenTelemetry or contributing to the project.
+
+Below you will find a list of talks and other activities related to
+OpenTelemetry.
+
+## Kubecon Talks and Maintainer Sessions
+
+- **[Jaeger: The Future with OpenTelemetry and Metrics](https://sched.co/1HySf)**<br>
+  by Pavol Loffay, Red Hat & Jonah Kowall, Aiven<br> Wednesday, April 19 •
+  11:55 - 12:30
+- **[Using OpenTelemetry for Application Security, with a Real Life Example](https://sched.co/1HyYT)**<br>
+  by Ron Vider, Oxeye<br> Wednesday, April 19 • 11:55 - 12:30
+- **[Understand Systems with OpenTelemetry: A Hybrid Telemetry Data Backend](https://sched.co/1Hyb2)**<br>
+  by Ran Xu, Huawei & Xiaochun Yang, Northeastern University<br> Wednesday,
+  April 19 • 14:30 - 15:05
+- **[OTel Me About Metrics: A Metrics 101 Crash Course](https://sched.co/1Hya1)**<br>
+  by Reese Lee, New Relic<br> Wednesday, April 19 • 17:25 - 18:00
+- **[OpenTelemetry: Using Unified Semantics to Drive Insights + Project Update](https://sched.co/1HyS5)**<br>
+  by Morgan McLean, Splunk, Alolita Sharma, Apple, Daniel Dyla, Dynatrace, & Ted
+  Young, Lightstep<br> Thursday, April 20 • 16:30 - 17:05
+- **[Tutorial: Exploring the Power of OpenTelemetry on Kubernetes](https://sched.co/1HyZ3)**<br>
+  by Pavol Loffay, Benedikt Bongartz & Yuri Oliveira Sa, Red Hat, Severin
+  Neumann, Cisco & Kristina Pathak, LightStep<br> Friday, April 21 • 16:00 -
+  17:30
+
+## Co-located Events
+
+The [OpenTelemetry project meeting](https://sched.co/1JWS7) is to meet and
+network with the OpenTelemetry maintainers and core contributors. This meeting
+will be held 13:00 - 15:00 on Tuesday April 18, 2023.
+
+[Observability Day][] _fosters collaboration, discussion, and knowledge sharing
+of cloud-native observability projects_. This event will be held on April 18,
+2023 from 9:00 - 17:00. There will be several sessions on OpenTelemetry as well.
```

Comment:
I think it would be worth pointing out that for the project meeting, one does _not_ need an all-access pass (right?) but for the Observability Day they would and those are already sold out as far as I know.
The ticket structure was recently changed which has led to some confusion already.

Reply:
```suggestion
The [OpenTelemetry project meeting](https://sched.co/1JWS7) is to meet and
network with the OpenTelemetry maintainers and core contributors. This meeting
will be held 13:00 - 15:00 on Tuesday April 18, 2023.
_This meeting can be attended with the standard in-person pass and does not require an "all-access" pass._

[Observability Day][] _fosters collaboration, discussion, and knowledge sharing
of cloud-native observability projects_. This event will be held on April 18,
2023 from 9:00 - 17:00. There will be several sessions on OpenTelemetry as well.
_This co-located event is not included in the general in-person pass but only in the "in-person all-access" pass. See the [Kubecon website](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/register/) for more details._
```
^ at least that's my understanding

Reply:
You're right @arminru! That's a good call out